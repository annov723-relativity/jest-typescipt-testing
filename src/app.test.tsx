import React from "react"
import { render } from "@testing-library/react"
import { CartItemsDisplay } from "./components/CarItemsDisplay"
import { CartItem } from "./etc/get-items-total"

describe("The Cart Item Display", () => {
    it("Should match the snapshot", () => {
        const specItems : CartItem[] = [
            {name: "item1", cost: 1},
            {name: "item2", cost: 2}
        ]

        const { container } = render(<CartItemsDisplay shoppingCartItems={specItems} />)
        console.info(container)

        expect(container).toMatchSnapshot()
    })
})