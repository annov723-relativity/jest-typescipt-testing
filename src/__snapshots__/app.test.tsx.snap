// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Cart Item Display Should match the snapshot 1`] = `
<div>
  <div>
    item1
     - 
    1
     (SALE!)
  </div>
  <div>
    item2
     - 
    2
     (SALE!)
  </div>
</div>
`;
